modular_skyrat/modules/customization/strings/names/phonetic_alphabet_numbers.txt

```diff
@@ -0,0 +1,10 @@
+Zero
+One
+Two
+Three
+Four
+Five
+Six
+Seven
+Eight
+Niner
```

Comment:
After thinking this through, this *really* did feel better when I was thinking of it rather than when it's executed; I'll probably come up with a better system of RNG numbers at a later date.